libs/ui-lib/lib/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx

```diff
@@ -86,13 +86,17 @@ export const LimitedSupportedCluster = ({
   <TextContent>
     {showVersionWarning && (
       <Text>
-        <InfoCircleIcon size="sm" color="var(--pf-global--info-color--100)" />
+        <Icon size="sm" color="var(--pf-global--info-color--100)">
```

Comment:
Have these variables not been renamed with the `pf-v5` prefix?
It may be safer to use the named color variable, or even better, a `variant` prop.

Same for the other icon in line 97.

Reply:
Good catch